data/web/inc/lib/vendor/php-mime-mail-parser/php-mime-mail-parser/compile_mailparse.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Why is there a .sh file in the web directory? This makes no sense. Also the file uses paths that have nothing to do with Mailcow? `/home/travis/.phpenv/versions/7.3.2/lib/php/extensions/no-debug-zts-20180731/` .